cocos/platform/CCFileUtils.cpp

```diff
@@ -1261,21 +1241,12 @@ void FileUtils::listFilesRecursively(const std::string& dirPath, std::vector<std
                     // Error getting file
                     break;
                 }
+                std::string fileName = file.name;
 
-#ifdef UNICODE
-                std::wstring path = file.path;
-                length = WideCharToMultiByte(CP_UTF8, 0, &path[0], (int)path.size(), NULL, 0, NULL, NULL);
-                std::string filepath;
-                if (length > 0)
-                {
-                    filepath.resize(length);
-                    WideCharToMultiByte(CP_UTF8, 0, &path[0], (int)path.size(), &filepath[0], length, NULL, NULL);
-                }
-#else
-                std::string filepath = file.path;
-#endif
-                if (file.name[0] != '.')
+                // Need check full name file on "." and "..", otherwise exclude file and folder begin '.'
+                if (fileName != "." && fileName != "..")
```

Comment:
Yep, i know the meaning. How about just remove the comment, because the codes is clear and i am confused when check the comment.

`.git` and `.vs` is hidden folder in Mac and linux like system.